db/db_tunables.c

```diff
@@ -79,6 +79,7 @@ extern int gbl_max_lua_instructions;
 extern int gbl_max_sqlcache;
 extern int __gbl_max_mpalloc_sleeptime;
 extern int gbl_mem_nice;
+extern int gbl_verbose_net;
 extern int gbl_netbufsz;
```

Comment:
Is the "verbose_net" tunable actually checked somewhere?  I don't see where it is used in this pull request.